src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -248,6 +248,18 @@ public partial class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.Reso
             HelpMessage = "Set this flag to true to get a boost on the performance target of the disk deployed, see here on the respective performance target. This flag can only be set on disk creation time and cannot be disabled after enabled.")]
         public bool? PerformancePlus { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Required if createOption is CopyFromSanSnapshot. this is the ARM id of the source elastic san volume snapshot.")]
+        public string ElasticSanResourceId { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Setting this property to true improves reliability and performance of data disks that are frequently (more than 5 times a day) by detached from one virtual machine and attached to another. This property should not be set for disks that are not detached and attached frequently as it causes the disks to not align with the fault domain of the virtual machine.")]
+        public bool? OptimizedForFrequentAttach { get; set; }
```

Comment:
PS prefers to use SwitchParameters instead of bools.

Reply:
This is right. However, strangely for this cmdlet, all the boolean properties are set with a boolean parameter. so this should be okay staying consistent.